spec/services/search/vacancy_sort_spec.rb

```diff
@@ -46,6 +46,18 @@
         expect(subject.options.map(&:sort_by)).not_to include("relevance")
       end
     end
+
+    context "when the sort option `expires_at` is selected" do
+      let(:sort_by) { "expires_at" }
+
+      it "sorts by publish_on" do
+        expect(subject.sort_by).to eq("expires_at")
+      end
+
+      it "has order 'desc'" do
+        expect(subject.order).to eq("asc")
```

Comment:
:raising_hand: Either the test description or the assertion is wrong here (as they contradict each other).